packages/beacon-node/src/chain/validation/attestation.ts

```diff
@@ -414,9 +414,9 @@ async function validateAttestationNoSignatureCheck(
   if (!isForkPostElectra(fork)) {
     // The validity of aggregation bits are already checked above
     assert.notNull(aggregationBits);
-    assert.notNull(committeeValidatorIndex);
+    assert.notNull(validatorCommitteeIndex);
 
-    validatorIndex = committeeValidatorIndices[committeeValidatorIndex];
+    validatorIndex = committeeValidatorIndices[validatorCommitteeIndex];
```

Comment:
Lol my brain is exploding. Good thing you keep `validatorIndex` or it is going to be

`committeeValidatorIndex = committeeValidatorIndices[validatorCommitteeIndex];`

Reply:
I think `committeeValidatorIndex` (singular) does not make sense, should always call this `validatorIndex`